packages/react-next/src/components/TextField/MaskedTextField/MaskedTextField.test.tsx

```diff
@@ -13,6 +13,12 @@ describe('MaskedTextField', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  isConformant({
+    Component: MaskedTextField,
+    displayName: 'MaskedTextField',
+    disabledTests: ['has-top-level-file', 'exported-top-level'],
```

Comment:
This does appear to be exported (at least in OUFR--if it's not in react-next we should fix that)
```suggestion
    disabledTests: ['has-top-level-file'],
```